.github/workflows/sonarqube.yml

```
@@ -0,0 +1,70 @@
+
+      - name: Cache Sonar
+        uses: actions/cache@v4
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+
+      - name: Run SonarQube
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)